### PR TITLE
Professional website

### PR DESCRIPTION
**Pull request by rdbogle**
Oohhh yeahh...

### CODE REVIEW

File: index2.html [hunk 1]
```diff
@@ -0,0 +1,177 @@
+<!doctype html>
```
Comment (evertheylen) [inline index2.html:1]: Generally speaking it's better to let git do it's thing and remove old files :) But I can see why you did it with the major rewrite

File: style2.css [hunk 2]
```diff
@@ -0,0 +1,175 @@
+
```
Comment (evertheylen) [inline style2.css:1]: Same thing with this file (one could even say, _especially_ with this file and all others that are now potentially redundant)

File: index.html [hunk 3]
```diff
@@ -1,176 +1,780 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="">
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Poppins&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
 
-    <link rel="stylesheet" href="style.css">
+    <title> Quanteco foodprint</title>
```
Comment (evertheylen) [inline index.html:15]: Drop the preceding space

File: index.html [hunk 4]
```diff
@@ -1,176 +1,780 @@
...
+        <div class="col-12">
+          <nav class="main-nav">
+            <!-- ***** Logo Start ***** -->
+            <a href="index.html" class="logo">
+              <img src="assets/images/qe/quanteco2.png" alt="Quanteco" style="width: 130px;">
+            </a>
+            <!-- ***** Logo End ***** -->
+            <!-- ***** Menu Start ***** -->
+            <ul class="nav">
+              <li class="scroll-to-section"><a href="#top" class="active">Login</a></li>
```
Comment (evertheylen) [inline index.html:65]: I would very much remove the focus on "Login". There is already a big shiny green "To The Alpha" button, so why bother with "Login"?

File: index.html [hunk 5]
```diff
@@ -1,176 +1,780 @@
...
+            </a>
+            <!-- ***** Menu End ***** -->
+          </nav>
         </div>
+      </div>
+    </div>
+  </header>
+  <!-- ***** Header Area End ***** -->
+  
+  <div id="modal" class="popupContainer" style="display:none;">
```
Comment (evertheylen) [inline index.html:83]: I feel like this whole `div` is just for login and can therefore be removed?

File: index.html [hunk 6]
```diff
@@ -1,176 +1,780 @@
...
+    <div class="container" style="margin-bottom: 140px;">
+      <div class="row">
+        <div class="col-lg-12">
+          <div class="row">
+            <div class="col-lg-6 align-self-center">
+              <div class="left-content show-up header-text wow fadeInLeft" data-wow-duration="1s" data-wow-delay="1s">
+                <div class="row">
+                  <div class="col-lg-12">
+                    <h2>Login to foodprint</h2>
+                    <p>The recipe and diet planning app with full nutritional and ecological data. Available on Android, iOS and in browser! Register now and eat well with ease. Free forever, we promise!</p>
```
Comment (evertheylen) [inline index.html:179]: "Available on Android, iOS and in _any modern_ browser" or something like it

File: index.html [hunk 7]
```diff
@@ -1,176 +1,780 @@
...
+                    <div class="white-button first-button scroll-to-section">
+                      <a href="https://apps.apple.com/lt/app/cooking-fever/id714796093" target="_blank">App store <i class="fab fa-apple"></i></a>
+                    </div>
+                    <div class="white-button scroll-to-section">
+                      <a href="https://play.google.com/store/apps/details?id=com.nordcurrent.canteenhd" target="_blank"> Google Play<i class="fab fa-google-play"></i></a>
+                    </div>
+                  <br>
+                  -->
+                    <div class="white-button scroll-to-section">
+                      <a href="https://staging.quanteco.org/search" target="_blank"> Login Online</a>
```
Comment (evertheylen) [inline index.html:192]: Maybe copy the "To The Alpha" button here or so

File: index.html [hunk 8]
```diff
@@ -1,176 +1,780 @@
...
+    </div>
+  </div>
+  <div id="services" class="services section">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-8 offset-lg-2">
+          <div class="section-heading  wow fadeInDown" data-wow-duration="1s" data-wow-delay="0.5s">
+            <h4>Eat with your heart</h4>
+              
+            <img src="assets/images/heading-line-dec.png" alt="">
```
Comment (evertheylen) [inline index.html:216]: Tbh I don't really get the appeal of the weird two blue lines, looks better without imho (everywhere)

File: index.html [hunk 9]
```diff
@@ -1,176 +1,780 @@
...
+              <a href="#">Read More <i class="fa fa-arrow-right"></i></a>
+            </div>
+          -->
+          </div>
+        </div>
+        <div class="col-lg-3">
+          <div class="service-item second-service">
+            <div class="icon"></div>
+            <h4>Find recipes</h4>
+            <p>Find recipes suited to all levels, from student to master chef, and check their environmental impact. Coming soon... make your own recipes! </p>
```
Comment (evertheylen) [inline index.html:240]: "Soon, you'll be able to make your own recipes!"

File: index.html [hunk 10]
```diff
@@ -1,176 +1,780 @@
...
+              <a href="#">Read More <i class="fa fa-arrow-right"></i></a>
+            </div>
+          -->
+          </div>
+        </div>
+        
+        <div class="col-lg-3">
+          <div class="service-item fourth-service">
+            <div class="icon"></div>
+            <h4>Count calories, count CO2 too!</h4>
```
Comment (evertheylen) [inline index.html:264]: `CO<sub>2</sub>`

File: index.html [hunk 11]
```diff
@@ -1,176 +1,780 @@
...
+            <img src="assets/images/heading-line-dec.png" alt="">
+            <p> Choose from our growing library of recipes. See their environmental impact and nutritional values. Track your nutrition and CO<sub>2</sub> output on a daily basis. Together we can meet our climate goals.
+            </div>
+        </div>
+      </div>
+    </div>
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-3">
+          <div class="service-item first-service">
```
Comment (evertheylen) [inline index.html:225]: The leaf that appears in all 4 cards makes the text hard to read. Maybe make it a lot softer so the text can remain black?

File: index.html [hunk 12]
```diff
@@ -1,176 +1,780 @@
...
+
+    <!-- Additional CSS Files -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <link rel="stylesheet" href="assets/css/templatemo-chain-app-dev.css">
+    <link rel="stylesheet" href="assets/css/animated.css">
+    <link rel="stylesheet" href="assets/css/owl.css">
+
+  </head>
 
 <body>
```
Comment (evertheylen) [inline index.html:37]: The text of the main paragraphs is very fine and hard to read. Could use just a little more thickness and a little darker shade of grey.

File: index.html [hunk 13]
```diff
@@ -1,176 +1,780 @@
...
+            <p>At Quanteco we are working towards a future where we can track our entire carbon footprint online, and live our lives mindful of the effects of our lifestyle. We believe that with an understanding of our true impact as individuals, we can harness our competitive and social potential to solve the climate crisis.
+            </br>
+              Using the science of life cycle analysis, we strip away the layers of industry between farm and plate, so that normal people can make decisions that improve the agricultural techniques of the world.</p>
+      
+            </p>
+          <div class="row">
+            <div class="col-lg-6">
+              <div class="box-item">
+                <h5> Robert - CEO </h5>
+               <p><img class="teamimage" src="assets/images/qe/robert.png" style="height: 267px; width: 175px;" alt="Robert"> <br></p>
```
Comment (evertheylen) [inline index.html:293]: We need more professional, consistent, and equally sized images :)

File: index.html [hunk 14]
```diff
@@ -1,176 +1,780 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="">
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Poppins&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
 
-    <link rel="stylesheet" href="style.css">
```
Comment (evertheylen) [inline index.html:24]: Do we no longer have our own main stylesheet? All styles are from the template then? I'm confused

File: index.html [hunk 15]
```diff
@@ -1,176 +1,780 @@
...
+              </div>
+            </div>
+            <div class="col-lg-6">
+              <div class="box-item">
+                <h4><a href="#">Co. Development</a></h4>
+                <p>Lorem Ipsum Text</p>
+              </div>
+            </div>
+            -->
+             <p> If our goal appeals to you and you would like to advance our cause, please consider <a rel="nofollow" href="https://paypal.me" target="_blank">donating to us via PayPal</a>. Donations over €50 will provide access to the Quanteco discord server where we discuss app development and upcoming features. Thank you.
```
Comment (evertheylen) [inline index.html:329]: Uuuhm we should discuss this right (not that I strongly disagree). Also the paypal link is empty, is this really publish-worthy?

**Review by evertheylen — CHANGES_REQUESTED** [review]
Looks good, a few comments of course :) The donation thing (while it looks minor) is what blocks me from throwing this on master

File: index.html (hunk 15 above)
Comment (rdbogle) [reply index.html:329]: Agree. Definitely not publish worthy, just a draft

File: index.html (hunk 14 above)
Comment (rdbogle) [reply index.html:24]: Everything from the template

File: index.html (hunk 13 above)
Comment (rdbogle) [reply index.html:293]: Let's take them then!

File: index.html (hunk 5 above)
Comment (rdbogle) [reply index.html:83]: Just keeping it for if and when it's useful. Was planning to leave these redundancies there for a few months until we have actual users

File: index2.html (hunk 1 above)
Comment (rdbogle) [reply index2.html:1]: Yeah inefficiency due to git inexperience

**Comment by rdbogle** [comment]
Agree with pretty much everything. Tomorrow I'll try and make it deployment worthy

**Pull request merged**